docs/TROUBLESHOOTING.md

```diff
@@ -102,6 +102,16 @@ ControllerMode = dual
 JustWorksRepairing = confirm
 ```
 
+#### Xbox Wireless controller
+
+The newest wireless controllers from Microsoft (Xbox One and Xbox Series X/S) are known to cause a reconnect loop and not pairing with Bluez. There are some specific workarounds :
```

Comment:
Please add line wraps at 80 or 100 columns for newly added lines.

Reply:
Please don't [plenk](https://en.wikipedia.org/wiki/Plenken): Remove extraneous blanks in front of punctuation.

Reply:
Yep, sorry about that. I was doing the commit on Github, now I can format it on my computer.

Sorry for the punctation, I'm too used to french typography...